package/widevine/widevine.mk

```diff
@@ -30,22 +32,14 @@ define WIDEVINE_CONFIGURE_CMDS
        ./build.py arm)
 endef
 
-define WIDEVINE_BUILD_CMDS
-        export WIDEVINE_TARGET_DIR="$(TARGET_DIR)";\
-        export WIDEVINE_STAGING_DIR="$(STAGING_DIR)";\
-        make CROSS_COMPILE="$(TARGET_CROSS)" \
-        CXXFLAGS="$(TARGET_CXXFLAGS)" \
-        LDFLAGS="$(TARGET_LDFLAGS)  -L$(STAGING_DIR)/usr/lib" -C $(@D) -f Makefile
-endef
-
 define WIDEVINE_INSTALL_TARGET_CMDS
         cp $(@D)/out/arm/Debug/widevine_ce_cdm_unittest $(TARGET_DIR)/usr/bin
-        cp $(@D)/out/arm/Debug/lib.target/lib*.so $(TARGET_DIR)/usr/lib/
+        cp $(@D)/out/arm/Debug/lib*/lib*.so $(TARGET_DIR)/usr/lib/
 endef
 
 define WIDEVINE_INSTALL_STAGING_CMDS
         cp $(@D)/out/arm/Debug/widevine_ce_cdm_unittest $(STAGING_DIR)/usr/bin
-        cp $(@D)/out/arm/Debug/lib.target/lib*.so $(STAGING_DIR)/usr/lib/
+        cp $(@D)/out/arm/Debug/lib*/lib*.so $(STAGING_DIR)/usr/lib/
```

Comment:
Why change to copying the static archives? I thought we only needed the shared lib in the target dir

Reply:
Not copying static achieve, only *.so is copying. 
using make build system, libwidevine_ce_cdm_shared.so is getting generated in out/arm/Debug/lib.target/, but for ninja it is in out/arm/Debug/lib/ so that this changed to lib*/lib*.so

Reply:
Thanks for comments.